kernel/src/device/tty/termio.rs

```diff
@@ -127,9 +133,11 @@ pub enum C_CFLAGS_BAUD {
 }
 
 bitflags! {
+    /// The local flags; `c_lflags` bits in Linux.
     #[repr(C)]
     #[derive(Pod)]
-    pub struct C_LFLAGS: u32 {
+    pub(super) struct CLocalFlags: u32 {
```

Comment:
Just curious: I didn't find any documentation explaining that the l in c_lflags stands for 'Local.' Is there any reference available for this?

Reply:
[man pages](https://www.man7.org/linux/man-pages/man3/termios.3.html):
```
   The termios structure
       Many of the functions described here have a termios_p argument
       that is a pointer to a termios structure.  This structure contains
       at least the following members:

           tcflag_t c_iflag;      /* input modes */
           tcflag_t c_oflag;      /* output modes */
           tcflag_t c_cflag;      /* control modes */
           tcflag_t c_lflag;      /* local modes */
           cc_t     c_cc[NCCS];   /* special characters */
```